packages/components/src/SubHeaderBar/InputTitleSubHeader/InputTitleSubHeader.test.js

```diff
@@ -69,6 +66,12 @@ describe('TitleSubHeader', () => {
 		expect(wrapper.getNode().props.className).toEqual('tc-subheader-details-text-subtitle');
 		expect(wrapper.getNode().props.children).toEqual('mySubTitle');
 	});
+	it('should render with with edit pencil', () => {
+		const wrapper = shallow(<TitleSubHeader {...defaultProps} subTitle="mySubTitle" edit />).find(
+			'small',
+		);
+		expect(wrapper.find(Action)).toHaveLength(0);
+	});
 	it('should render an Action', () => {
 		const actionProps = {
```

Comment:
`actionProps ` is created but only used after to get `actionProps.onClick`.
Remove this object